test_scripts/Defects/7_1/1585_AlertManeuver_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -1,20 +1,20 @@
 ---------------------------------------------------------------------------------------------------
 -- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
 ---------------------------------------------------------------------------------------------------
--- Description: HMI responds with UNSUPPORTED_RESOURCE to Speak component of AlertManeuver
+-- Description: HMI responds with UNSUPPORTED_RESOURCE resultCode to Navigation.AlertManeuver component
 --
 -- Preconditions:
 -- 1) Clean environment
 -- 2) SDL, HMI, Mobile session started
--- 3) Registered app
--- 4) Activated app
+-- 3) App is registered
+-- 4) App is activated
 --
--- Steps: 
--- 1) Send AlertManeuver mobile RPC from app with parametr imageType = "STATIC", HMI responds with 
---    UNSUPPORTED_RESOURCE to Speak request
+-- Steps:
+-- 1) Send AlertManeuver mobile RPC from app with parameter imageType = "STATIC", HMI responds with
+--    UNSUPPORTED_RESOURCE to Navigation.AlertManeuver request
 --
 -- Expected:
--- 1) App receives AlertManeuver response, with WARNINGS result code and success=true
+-- 1) App receives AlertManeuver response, with { success = true, resultCode = "UNSUPPORTED_RESOURCE" }
```

Comment:
Description says app has to receive `UNSUPPORTED_RESOURCE` however in function `sendAlertManeuver()` expected `WARNINGS`